mage/query-modules/python/date.md

```diff
@@ -0,0 +1,143 @@
+---
+id: date
+title: date
+sidebar_label: date
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import RunOnSubgraph from '../../templates/_run_on_subgraph.mdx';
+
+export const Highlight = ({children, color}) => (
+<span
+style={{
+  backgroundColor: color,
+  borderRadius: '2px',
+  color: '#fff',
+  padding: '0.2rem',
+}}>
+{children}
+</span>
+);
+
+The `date` module provides various utilities to handle date and time operations within the Cypher Query Language. These functions can be used in conjunction with other Cypher expressions to handle date-related tasks such as formatting, parsing, comparisons, and arithmetic, thus enhancing the capabilities of Memgraph for managing time-based data.
+
+[![docs-source](https://img.shields.io/badge/source-date-FB6E00?logo=github&style=for-the-badge)](https://github.com/memgraph/mage/tree/main/python/date.py)
+
+| Trait               | Value                                                 |
+| ------------------- | ----------------------------------------------------- |
+| **Module type**     | <Highlight color="#FB6E00">**algorithm**</Highlight>  |
+| **Implementation**  | <Highlight color="#FB6E00">**C++**</Highlight>        |
+| **Graph direction** | <Highlight color="#FB6E00">**directed**</Highlight>/<Highlight color="#FB6E00">**undirected**</Highlight> |
+| **Edge weights**    | <Highlight color="#FB6E00">**weighted**</Highlight>/<Highlight color="#FB6E00">**unweighted**</Highlight> |
+| **Parallelism**     | <Highlight color="#FB6E00">**sequential**</Highlight> |
+
+### Procedures
+
+### `format(time, unit, format, timezone)`
+
+Returns a string representation of time value using the specified unit, specified format, and specified time zone.
+
+#### Input:
+
+- `time: int` ➡ time which has passed since the Unix epoch
+- `unit: str (default="ms")` ➡ unit of the given time
+- `format: str (default="%Y-%m-%d %H:%M:%S %Z")` ➡ pattern to be formatted to
+- `timezone: str (default="UTC")` ➡ timezone to be used
+
+:::info
+
+The unit parameter supports the following values:
+- "ms" for milliseconds
+- "s" for seconds
+- "m" for minutes
+- "h" for hours
+- "d" for days
+
+:::
+
+:::info
+
+The format parameter supports values defined under [Python strftime format codes](https://docs.python.org/3/library/datetime.html#strftime-and-strptime-format-codes).
+
+:::
+
+:::info
+
+The timezone parameter can be specified with the database TZ identifier (text) name, as listed for [timezones](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones).
+
+:::
+
+#### Output:
```

Comment:
IF you have these many info boxes they kind of start losing meaning...  :)

It's just regular text and info about the module

Reply:
I really like it this way because it draws attention and those infos are really important since the usage of procedures is different from neo4j so I wanted it to be highlighted